wpeverest/everest-forms
# Tweak/pdf-download-text

### All Submissions:






### Changes proposed in this Pull Request:



PDF download text option added to pdf setting.

### How to test the changes in this Pull Request:

1. GOTO PDF Setting.
2. Click 'Yes' on 'Download pdf after form submit' option.
3. Verify download text option below 'Download pdf after form submit'

### Types of changes:




### Other information:




### Changelog entry

> Tweak/pdf-download-text
